styles.css

```diff
@@ -18,9 +18,27 @@ aside{
   flex: 1;
   background:#ddd;
 }
-.main{
+
+main{
   flex: 3;
 }
+
+header>h1 {
+  margin: 0;
+}
+
+footer>h3,p {
+  margin:0;
+}
+
+aside>h2 {
+  margin:0;
+}
+
+main>h2,div {
+  margin:0;
+}
+
```

Comment:
:+1:

This is correct, although you could've also written these like the following to account for all direct children, using `header` as an example...

```css
header> * {
  margin: 0;
}
```